packages/hawtio/src/plugins/index.ts

```diff
@@ -17,6 +18,7 @@ import { springboot } from './springboot'
 export const registerPlugins: HawtioPlugin = () => {
   // Auth plugins should be loaded before other plugins
   keycloak()
+  oidc()
```

Comment:
Please also look at the line 34 and add `oidc` to the export there. Since we decided to export each plugin entry to users so that they can assemble their own hawtio console with selected builtin plugins. Some might want to use only `oidc` but not `keycloak` ;-)

Reply:
done